<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="147dda81fa38" failures="0" tests="2" name="TestBot.Appium.iOSSwipeDemo" time="13.995" errors="1" timestamp="2024-05-05T23:39:15 IST" skipped="0">
  <testcase classname="TestBot.Appium.iOSSwipeDemo" name="tearDown" time="3.273">
    <error message="Connection refused
Build info: version: &#039;4.20.0&#039;, revision: &#039;866c76ca80&#039;
System info: os.name: &#039;Mac OS X&#039;, os.arch: &#039;x86_64&#039;, os.version: &#039;14.4.1&#039;, java.version: &#039;11.0.23&#039;
Driver info: io.appium.java_client.ios.IOSDriver
Command: [3df50f83-1834-4836-82a3-0b75dce42c31, quit {}]
Capabilities {appium:automationName: XCuiTest, appium:databaseEnabled: false, appium:deviceName: iPhone 12, appium:javascriptEnabled: true, appium:locationContextEnabled: false, appium:networkConnectionEnabled: false, appium:platformVersion: 17.4, appium:takesScreenshot: true, appium:udid: A273E761-2538-4F20-95BD-1D0..., appium:wdaLaunchTimeout: 10000, appium:webStorageEnabled: false, platformName: IOS}
Session ID: 3df50f83-1834-4836-82a3-0b75dce42c31" type="org.openqa.selenium.WebDriverException">
      <![CDATA[org.openqa.selenium.WebDriverException: Connection refused
Build info: version: '4.20.0', revision: '866c76ca80'
System info: os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '14.4.1', java.version: '11.0.23'
Driver info: io.appium.java_client.ios.IOSDriver
Command: [3df50f83-1834-4836-82a3-0b75dce42c31, quit {}]
Capabilities {appium:automationName: XCuiTest, appium:databaseEnabled: false, appium:deviceName: iPhone 12, appium:javascriptEnabled: true, appium:locationContextEnabled: false, appium:networkConnectionEnabled: false, appium:platformVersion: 17.4, appium:takesScreenshot: true, appium:udid: A273E761-2538-4F20-95BD-1D0..., appium:wdaLaunchTimeout: 10000, appium:webStorageEnabled: false, platformName: IOS}
Session ID: 3df50f83-1834-4836-82a3-0b75dce42c31
at io.appium.java_client.remote.AppiumCommandExecutor.lambda$execute$2(AppiumCommandExecutor.java:248)
at java.base/java.util.Optional.orElseGet(Optional.java:369)
at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:248)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:518)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:591)
at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:307)
at org.openqa.selenium.remote.RemoteWebDriver.quit(RemoteWebDriver.java:433)
at TestBot.Appium.iOSBaseTest.tearDown(iOSBaseTest.java:68)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:393)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:326)
at org.testng.internal.invokers.TestMethodWorker.invokeAfterClassConfigurations(TestMethodWorker.java:229)
at org.testng.internal.invokers.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:215)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:131)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.net.ConnectException: Connection refused
at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:777)
at java.net.http/jdk.internal.net.http.PlainHttpConnection$ConnectEvent.handle(PlainHttpConnection.java:128)
at java.net.http/jdk.internal.net.http.HttpClientImpl$SelectorManager.handleEvent(HttpClientImpl.java:957)
at java.net.http/jdk.internal.net.http.HttpClientImpl$SelectorManager.lambda$run$3(HttpClientImpl.java:912)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.net.http/jdk.internal.net.http.HttpClientImpl$SelectorManager.run(HttpClientImpl.java:912)
]]>
    </error>
  </testcase> <!-- tearDown -->
  <testcase classname="TestBot.Appium.iOSSwipeDemo" name="iOSSwipeTest" time="10.722"/>
</testsuite> <!-- TestBot.Appium.iOSSwipeDemo -->
